lib/ast.js

```diff
@@ -1241,6 +1242,21 @@ var AST_True = DEFNODE("True", null, {
     value: true
 }, AST_Boolean);
 
+/* -----[ Yield ]----- */
+
+var AST_Yield = DEFNODE("Yield", "value is_star", {
+    $documentation: "A `yield` statement",
+    $propdoc: {
+        value: "[AST_Node?] the value returned or thrown by this statement; could be null",
+        is_star: "[Boolean] Whether this is a yield or yield* statement"
+    },
+    _walk: function(visitor) {
+        return visitor._visit(this, this.value && function(){
+            this.value._walk(visitor);
+        });
+    }
+}, AST_Statement);
```

Comment:
Just confirmed that all tests still pass with `AST_Yield` not being derived from a base class.